scripts.js

```diff
@@ -0,0 +1,95 @@
+/* eslint-disable no-plusplus */
```

Comment:
- [ ] As the first reviewer mentioned, kindly remove this `Eslint-disable` statement in this file. `Eslint` helps us to write standard `JavaScript` codes, using `Eslint-disable` statements will defeat this purpose. 👍 😎 

#### Hint: The error says you shouldn't use `index++` but you should use `index += 1`